css/properties/align-self.json

```
@@ -6,6 +6,7 @@
           "__compat": {
             "description": "Supported in Flex Layout",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/align-self",
+            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#propdef-align-self",
```

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#align-items-property",
```

Actually this is a case which makes a mess of an attempt to link to sections over the definitions, as this section is for both `align-items` and `align-self`.

---

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#propdef-align-self",
```

---

If we're going to link to the section, shouldn't this also link to https://drafts.csswg.org/css-flexbox/#align-items-property even though the ID makes it look wrong?

---

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#align-items-property",
```